src/Controller/CaseController.php

```diff
@@ -38,10 +42,60 @@ public function summary(CaseEntity $case): Response
     /**
      * @Route("/{id}/information", name="case_information", methods={"GET"})
      */
-    public function information(CaseEntity $case): Response
+    public function information(CaseEntity $case, ComplaintCategoryRepository $categoryRepository): Response
```

Comment:
This controller method shows the CaseEntity's data, and should follow the convention in other Symfony projects. Hence this method should be renamed to `public function show` and the route should be `@Route("/{id}")` and the route name should be `case_show`

Reply:
Renamed route and function accordingly.